.drone/drone.yml

```diff
@@ -49,7 +49,6 @@ steps:
       from_secret: docker_username
   when:
     ref:
-    - refs/heads/main
```

Comment:
this is fine... now we need a way to push builds from main to docker so that we can use them in staging...

use a different image name?

Reply:
Good suggestion - thanks for the review, and for creating that github issue 👍